libraries/ui-library/src/components/six-datepicker/six-date-formats.tsx

```diff
@@ -10,6 +10,11 @@ export enum SixDateFormats {
   DDMMYY_SLASH = 'dd/mm/yy',
   YYMMDD_SLASH = 'yy/mm/dd',
 
+  DDMMYYY_DOT_TIME_MINUTES = 'dd.mm.yyyy hh:MM',
```

Comment:
If I use this, the timpicker part is still displaying the seconds:
![image](https://github.com/six-group/six-webcomponents/assets/1475511/2476b8f3-f2f1-4364-a206-092eb3284c33)